### ISSUE TITLE
Reaction tooltips breaking

### ISSUE THREAD

**Issue by Mottie** [ZenHubIO/support#623]
GitHub sets their multiline tooltips css definition to `display: table-cell`:

```css
.tooltipped-multiline:hover::after, .tooltipped-multiline:active::after, .tooltipped-multiline:focus::after {
    display: table-cell;
}
```

And the ZenHub extension adds the following:

```css
*, ::before, ::after {
    box-sizing: border-box;
}
```

Which breaks the multiline setting, at least in Chrome, and makes the reaction tooltips very narrow... Hover over the reactions below to see it happen.

**Comment by azenMatt** [comment]
Haha - no way. I thought this was a GH screw up! I was wondering why it was taking them so long to fix! 

Thanks @Mottie - we will get this fixed!

**Comment by azenMatt** [comment]
Hey @Mottie, this is now fix on 2.15.94 which we just released and should be hitting your browser soon! 

![image](https://cloud.githubusercontent.com/assets/7584089/14182260/60ad18d6-f71f-11e5-8619-522a3164afff.png)

**Comment by Mottie** [comment]
Awesome! Thanks!
